hms-app/src/pages/Archive.tsx

```diff
@@ -3,22 +3,26 @@ import HackathonDetails from '../components/card-details/HackathonDetails'
 import HackathonSelectDropdown from '../components/HackathonSelectDropdown'
 import { Title } from '@mantine/core'
 
-export default function Archive() {
-  const [selectedHackweek, setSelectedHackweek] = useState('')
+enum Enum {
+  Archive = 'ARCHIVE',
+  Header = 'HEADER',
+}
 
-  const setHackathonID = (hackthonID: string) => {
-    setSelectedHackweek(hackthonID)
-  }
+export default function Archive() {
+  const [selectedHackathonId, setSelectedHackathonId] = useState('')
```

Comment:
likely we should set the value to something if it has no value :thinking:

Reply:
in best case to the last loaded hackathon but still struggling with saving it